homeassistant/components/recorder/websocket_api.py

```diff
@@ -28,3 +30,23 @@ async def ws_validate_statistics(
         hass,
     )
     connection.send_result(msg["id"], statistic_ids)
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "recorder/clear_statistics",
+        vol.Required("statistic_ids"): [str],
+    }
+)
+@websocket_api.async_response
+async def ws_clear_statistics(
```

Comment:
There is nothing async here. 
```suggestion
@callback
def ws_clear_statistics(
```